.github/workflows/master-e2e.yaml

```diff
@@ -222,7 +213,7 @@ jobs:
           echo "K3s on Rancher Manager: ${{ env.INSTALL_K3S_VERSION }}" >> ${GITHUB_STEP_SUMMARY}
           echo "K8s version deployed on the cluster(s): ${{ inputs.k8s_version_to_provision }}" >> ${GITHUB_STEP_SUMMARY}
   delete-runner:
-    if: ${{ always() && needs.create-runner.result == 'success' && inputs.destroy_runner == true }}
+    if: ${{ always() && inputs.destroy_runner == true }}
```

Comment:
It will, but that's exactly my goal - there are some cases when the creation of vm can fail (for eg public ip not assigned or so) but the VM is running - this change will always delete the VM (and secrets) even if the VM is not fully operational.

Reply:
And you can still use "keep vm" variable when triggering the action.

Reply:
ok. I was afraid that in the case that for some reason the runner creation was not even executed (ie some outage) the delete runner may be hanging because it did not find anything. But I guess if this does not solve it we may find another way. Ok approving.